packages/manager/modules/pci/src/projects/project/storages/databases/database/replications/add-edit/add-edit.routing.js

```diff
@@ -0,0 +1,40 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider
+    .state(
+      'pci.projects.project.storages.databases.dashboard.replications.add',
+      {
+        url: '/add',
+        component: 'ovhManagerPciProjectDatabaseReplicationsAddEdit',
```

Comment:
Related to this https://github.com/ovh/manager/pull/5749/files#r742135137

Reply:
Done here: https://github.com/ovh/manager/pull/5749/commits/97ac6df6312f0f05dc5b40170c4365961e141283